testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java

```diff
@@ -689,4 +688,104 @@ public void verifyFT77()
                 expectedJsonPattern, ADMIN_USERNAME, ADMIN_PASSWORD));
     }
 
+    @Test
+    public void verifyFT76() throws IOException, InterruptedException {
+        /*
+         * To check that only message for Active Pack should be returned from
+         * inbox after 7 days of user's subscription gets completed for 72Weeks
+         * Pack while user is subscribed for both Pack.
+         */
+        setupData();
```

Comment:
setupData() is called automatically on line 113 before each test, it's not needed here.